src/main/java/com/statful/client/framework/springboot/config/SpringbootClientConfiguration.java

```diff
@@ -0,0 +1,77 @@
+package com.statful.client.framework.springboot.config;
+
+import com.statful.client.domain.api.StatfulClient;
+import com.statful.client.framework.springboot.writer.StatfulMetricWriter;
+import org.springframework.boot.actuate.autoconfigure.ExportMetricWriter;
+import org.springframework.boot.actuate.endpoint.MetricsEndpoint;
+import org.springframework.boot.actuate.endpoint.MetricsEndpointMetricReader;
+import org.springframework.boot.actuate.metrics.writer.MetricWriter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+@ConfigurationProperties(prefix = "statful.client.springboot")
+@ConditionalOnBean(value = {StatfulClient.class, MetricWriter.class})
+public class SpringbootClientConfiguration {
+
+    private Metrics metrics;
+
+    public Metrics getMetrics() {
+        return metrics;
+    }
+
+    public void setMetrics(Metrics metrics) {
+        this.metrics = metrics;
+    }
+
+    public static class Metrics {
+        private List<Tags> tags = new ArrayList<>();
+
+        public List<Tags> getTags() {
+            return tags;
```

Comment:
you might want to consider returning a copy of this list `new ArrayList<>(tags);` to avoid external manipulation of the list, which would be even nicer if Tags were immutable